modules/govuk/manifests/node/s_db_admin.pp

```diff
@@ -150,7 +149,8 @@
   class { '::govuk::apps::policy_publisher::db': } ->
   class { '::govuk::apps::publishing_api::db': } ->
   class { '::govuk::apps::service_manual_publisher::db': } ->
-  class { '::govuk::apps::support_api::db': }
+  class { '::govuk::apps::support_api::db': } ->
+  class { '::govuk::apps::content_publisher::db': }
```

Comment:
Is the ordering necessary? Can also do

```
class { [
    '::govuk::apps::support_api::db',
    '::govuk::...',
  ]: }
```

Reply:
Yeah, we don't think it is. We thought we'd avoid breaking anything else, for now, though ;-)